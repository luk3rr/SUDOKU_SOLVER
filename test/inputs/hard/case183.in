000001000 060030091 000000700 040207000 700460020 900010000 670080009 300006270 000105038 
