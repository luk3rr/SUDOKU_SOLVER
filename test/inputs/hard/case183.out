Solving the following grid:
0 0 0 | 0 0 1 | 0 0 0 
0 6 0 | 0 3 0 | 0 9 1 
0 0 0 | 0 0 0 | 7 0 0 
------+-------+------
0 4 0 | 2 0 7 | 0 0 0 
7 0 0 | 4 6 0 | 0 2 0 
9 0 0 | 0 1 0 | 0 0 0 
------+-------+------
6 7 0 | 0 8 0 | 0 0 9 
3 0 0 | 0 0 6 | 2 7 0 
0 0 0 | 1 0 5 | 0 3 8 


Solution found :')

5 8 9 | 7 2 1 | 3 6 4 
2 6 7 | 8 3 4 | 5 9 1 
1 3 4 | 6 5 9 | 7 8 2 
------+-------+------
8 4 3 | 2 9 7 | 1 5 6 
7 5 1 | 4 6 8 | 9 2 3 
9 2 6 | 5 1 3 | 8 4 7 
------+-------+------
6 7 5 | 3 8 2 | 4 1 9 
3 1 8 | 9 4 6 | 2 7 5 
4 9 2 | 1 7 5 | 6 3 8 

Algorithm: BFS
Total time: 2734 ms
Total expanded states: 440212
